test/letters/containers/AddressSection.unit.spec.jsx

```diff
@@ -3,8 +3,12 @@ import ReactTestUtils from 'react-dom/test-utils';
 import SkinDeep from 'skin-deep';
 import { expect } from 'chai';
 import _ from 'lodash';
+import sinon from 'sinon';
 
-import { AddressSection } from '../../../src/js/letters/containers/AddressSection.jsx';
+import { getFormDOM } from '../../util/schemaform-utils';
```

Comment:
We may consider moving this out of `schemaform-utils` since it's not _necessarily_ schemaform-specific. I just put it there because...it seemed like the place to put it at the time. :man_shrugging:

Reply:
Hm...yeah I had never used this for non-rjsf tests!